components/test_raftstore/src/util.rs

```diff
@@ -818,10 +818,15 @@ pub fn must_kv_read_equal(client: &TikvClient, ctx: Context, key: Vec<u8>, val:
     assert_eq!(get_resp.take_value(), val);
 }
 
-// TODO: replace the redundant code
-pub fn complete_data_commit(client: &TikvClient, ctx: &Context, ts: u64, k: Vec<u8>, v: Vec<u8>) {
+pub fn complete_data_commit(
```

Comment:
I suggest using a more direct name.

```suggestion
pub fn write_and_read_key(
```